source/guide_wallabag.rst

```diff
@@ -0,0 +1,147 @@
+
+  * PHP_
+  * MySQL_
+  * domains_
+
+Prerequisites
+=============
+
+We're using PHP_ in the stable version 7.1:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '7.1'
+ [isabell@stardust ~]$
+
+.. include:: includes/my-print-defaults.rst
+
+Also, the domain you want to use for Lychee must be set up as well:
+
+.. include:: includes/web-domain-list.rst
+
+Installation & Configuration
+==============================
+
+First get the Wallabag source code from GitHub_:
+
+::
+
+  [isabell@stardust ~]$ git clone https://github.com/wallabag/wallabag.git ~/html
+  Cloning into '/home/isabell/wallabag'...
+  remote: Counting objects: 46655, done.
+  remote: Compressing objects: 100% (23/23), done.
+  remote: Total 46655 (delta 10), reused 20 (delta 9), pack-reused 46620
+  Receiving objects: 100% (46655/46655), 58.15 MiB | 20.81 MiB/s, done.
+  Resolving deltas: 100% (26623/26623), done.
+  [isabell@stardust ~]$
+
+Change to that folder and run ``make install``. During the installation process, you will be asked for quite a lot configuration settings. Bu you only need to set up the following informations:
+
+* ``database_name:`` <username>_wallabag - *replace <username> with the MySQL username*
+* ``database_user:`` put in the MySQL username that you looked up in the prerequisites
+* ``database_password:`` that is the MySQL password that you looked up in the prerequisites
+* ``domain_name:`` put in here your domain or subdomain like https://yourdomain
+* ``secret:`` type in any random string here, do not keep the defaul string!
+* ``twofactor_sender:`` choose a mailaddress that fits as sender
+* ``fosuser_registration:`` set this to false, otherwise anyone can register at your wallabag instance
+* ``from_email:`` choose a mailaddress that fits as sender (can be same as before)
+
+* ``Would you like to create a new admin user (recommended)?:`` yes
+* ``Username:`` admin *- or you can also choose any other name, that could increase security*
+* ``Password:`` *choose a good password here*
+* ``Email:`` set in a mailadress to use with this admin account
+
+.. code-block:: console
+ :emphasize-lines: 1,2,10,11,12,17,23,25,26,28,30,31,32,33
+
+  [isabell@stardust ~]$ cd ~/html
+  [isabell@stardust html]$ make install
+  [...]
+  Creating the "app/config/parameters.yml" file
+  Some parameters are missing. Please provide them.
+  database_driver (pdo_mysql): 
+  database_driver_class (null): 
+  database_host (127.0.0.1): 
+  database_port (null): 
+  database_name (wallabag): isabell_wallabag
+  database_user (root): isabell
+  database_password (null): MySuperSecretPassword
+  database_path (null): 
+  database_table_prefix (wallabag_):  
+  database_socket (null): 
+  database_charset (utf8mb4):  
+  domain_name ('https://your-wallabag-url-instance.com'): https://isabell.uber.space
+  mailer_transport (smtp): 
+  mailer_host (127.0.0.1): 
+  mailer_user (null): 
```

Comment:
This won't work …

Reply:
Mh true, it works on U6 and I thought I did successfully testing it on U7 but its not. I fixed it. 

I would have liked to link to a description how to set the email password in the U7 manual, but did not find any. 
Its just mentioned in the German dashboard//Datenblatt that you have to set the ssh password first.